tests/component/_task_modules/test_triggers.py

```diff
@@ -51,3 +67,59 @@ def test___arguments_provided___cfg_time_start_trig___no_errors(
     assert when_value.minute == trigger_time.minute
     assert when_value.second == trigger_time.second
     assert ai_voltage_field_daq_task.triggers.start_trigger.timestamp_timescale == timescale
+
+
+def test___start_trigger___wait_for_valid_timestamp___no_errors(
+    ai_voltage_field_daq_task: Task,
+):
+    ai_voltage_field_daq_task.timing.cfg_samp_clk_timing(1000)
+    ai_voltage_field_daq_task.triggers.start_trigger.timestamp_enable = True
+    ai_voltage_field_daq_task.start()
+
+    ai_voltage_field_daq_task.wait_for_valid_timestamp(TimestampEvent.START_TRIGGER)
```

Comment:
FYI these tests are all failing


```
tests\component\_task_modules\test_triggers.py:79: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
generated\nidaqmx\task.py:1036: in wait_for_valid_timestamp
    self._interpreter.wait_for_valid_timestamp(self._handle, timestamp_event.value, timeout)
generated\nidaqmx\_grpc_interpreter.py:3339: in wait_for_valid_timestamp
    return convert_timestamp_to_time(response.timestamp)
generated\nidaqmx\_grpc_time.py:49: in convert_timestamp_to_time
    dt = ht_datetime.fromtimestamp(seconds, timezone.utc)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

cls = <class 'hightime.datetime'>, t = -2082844800, tz = datetime.timezone.utc

    @classmethod
    def fromtimestamp(cls, t, tz=None):
        # NOTE: Does not support sub-microsecond values!
>       result = std_datetime.datetime.fromtimestamp(t, tz)
E       OSError: [Errno 22] Invalid argument
```

Reply:
they are failing because the test returns AbsoluteTime 0, which isnt supported in the libtime (because my fix to support negative timestamps is not in yet)

Reply:
Ah, ok :)